sherpa/fit.py

```diff
@@ -661,27 +671,31 @@ def sigmarej(self, statfunc, pars, parmins, parmaxes, statargs = (),
 
         maxiters = self.itermethod_opts['maxiters']
         if (type(maxiters) != int):
-            raise SherpaErr("'maxiters' value for sigma rejection method must be an integer")
+            raise SherpaErr(
+                "'maxiters' value for sigma rejection method must be an integer")
         if (maxiters < 1):
             raise SherpaErr("'maxiters' must be one or greater")
 
         hrej = self.itermethod_opts['hrej']
         if (type(hrej) != int and
-            type(hrej) != float):
-            raise SherpaErr("'hrej' value for sigma rejection method must be a number")
+                type(hrej) != float):
+            raise SherpaErr(
+                "'hrej' value for sigma rejection method must be a number")
         if (not (hrej > 0)):
-            raise SherpaErr("'hrej' must be greater than zero") 
+            raise SherpaErr("'hrej' must be greater than zero")
 
         lrej = self.itermethod_opts['lrej']
         if (type(lrej) != int and
-            type(lrej) != float):
-            raise SherpaErr("'lrej' value for sigma rejection method must be a number")
+                type(lrej) != float):
```

Comment:
Plus, there are better ways of checking if an object is a number (e.g. what if it was a long?).